src/box/lua/key_def.c

```diff
@@ -221,6 +226,24 @@ luaT_key_def_set_part(struct lua_State *L, struct key_part_def *part,
 	}
 	lua_pop(L, 1);
 
+	/* Set part->sort_order. */
+	lua_pushstring(L, "sort_order");
+	lua_gettable(L, -2);
+	if (!lua_isnil(L, -1)) {
+		size_t sort_order_len;
+		const char *sort_order = lua_tolstring(L, -1, &sort_order_len);
+		part->sort_order = strnindex(sort_order_strs, sort_order,
+					     sort_order_len, sort_order_MAX);
+		assert(part->sort_order != SORT_ORDER_UNDEF);
+		if (part->sort_order == sort_order_MAX) {
+			diag_set(IllegalParams, "unknown sort order");
```

Comment:
Nit: It is good to show a content of a wrong parameter in the error message, just like a compiler shows a line in the code that has a problem. It is not essential, but reduces a time needed to understand what is the problem.

See error messages for `type` and `collation` fields above for example.